src/main/webapp/academicAdminOffice/payments/showOperations.jsp

```diff
@@ -60,32 +61,12 @@
 		<li><html:link action="<%="/payments.do?method=showEvents&amp;personId=" + personId %>">
 			<bean:message bundle="ACADEMIC_OFFICE_RESOURCES" key="label.payments.currentEvents" />
 		</html:link></li>
-		<%--<li><html:link action="<%="/payments.do?method=showExternalEvents&amp;personId=" + personId %>">--%>
-			<%--<bean:message bundle="ACADEMIC_OFFICE_RESOURCES" key="label.scolarships.external" />--%>
-		<%--</html:link></li>--%>
-		<%--<li><html:link--%>
-			<%--action="<%="/receipts.do?method=showPaymentsWithoutReceipt&amp;personId=" + personId %>">--%>
-			<%--<bean:message bundle="ACADEMIC_OFFICE_RESOURCES" key="label.payments.paymentsWithoutReceipt" />--%>
-		<%--</html:link></li>--%>
-		<%--<li><html:link action="<%="/receipts.do?method=showReceipts&amp;personId=" + personId%>">--%>
-			<%--<bean:message bundle="ACADEMIC_OFFICE_RESOURCES" key="label.payments.receipts" />--%>
-		<%--</html:link></li>--%>
-		<%--<li><html:link action="<%="/payments.do?method=showEventsWithPaymentCodes&amp;personId=" + personId%>">--%>
-			<%--<bean:message bundle="ACADEMIC_OFFICE_RESOURCES" key="label.payments.paymentCodes" />--%>
-		<%--</html:link></li>--%>
-		<%--<li><html:link action="<%="/payments.do?method=showEventsWithPayments&amp;personId=" + personId%>">--%>
-			<%--<bean:message bundle="ACADEMIC_OFFICE_RESOURCES" key="label.payments.extract" />--%>
-		<%--</html:link></li>--%>
 	</ul>
 	<ul>
-		<li><html:link action="<%="/paymentsManagement.do?method=showEvents&personId=" + personId%>">
-			<bean:message bundle="ACADEMIC_OFFICE_RESOURCES" key="label.advanced.payments" />
-		</html:link></li>
+		<li>
+			<html:link href="<%= request.getContextPath() + "/accounting-management/" + username %>">
```

Comment:
This should be user's externalId instead of username, so it works with the changes in the AccountingController events method